Lombiq.Hosting.Tenants.Maintenance/StaggeredMigrations.cs

```diff
@@ -0,0 +1,57 @@
+using Lombiq.Hosting.Tenants.Maintenance.Constants;
+using Lombiq.Hosting.Tenants.Maintenance.Models;
+using OrchardCore.ContentFields.Fields;
+using OrchardCore.ContentFields.Settings;
+using OrchardCore.ContentManagement.Metadata;
+using OrchardCore.ContentManagement.Metadata.Settings;
+using OrchardCore.Data.Migration;
+using System.Threading.Tasks;
+
+namespace Lombiq.Hosting.Tenants.Maintenance;
+
+public sealed class StaggeredMigrations : DataMigration
+{
+    private readonly IContentDefinitionManager _contentDefinitionManager;
+
+    public StaggeredMigrations(IContentDefinitionManager contentDefinitionManager) =>
+        _contentDefinitionManager = contentDefinitionManager;
+
+    public async Task<int> CreateAsync()
+    {
+        await _contentDefinitionManager.AlterPartDefinitionAsync(nameof(StaggeredTenantWakeUpPart), part => part
+            .WithField<NumericField>(nameof(StaggeredTenantWakeUpPart.BatchSize), field => field
+                .WithDisplayName("Batch Size")
+                .WithSettings(new NumericFieldSettings
+                {
+                    Hint = "The number of tenants to be processed in each step or in case of parallel processing the" +
+                           " number of tenants started at once. If StaggeredTenantWakeUpOptions is set, these values will be ignored.",
```

Comment:
I just noticed this:
`If StaggeredTenantWakeUpOptions is set, these values will be ignored.`

Is this really the case? That's not how the settings UI usually works: If you have appsettings, then that's taken into account without having to go to the UI, but the values set in the UI should take higher precedence (and it should be initialized to appsettings, if exists), otherwise it's useless if you have anything set in appsettings.